scoring/update.html

```diff
@@ -76,12 +90,90 @@
 <rect height="120" width="120" stroke="#000" y="240" x="240" stroke-width="1" fill="#f4f3ff" />
 {% endblock %}
 
+{% block svg_size %}height="820" width="600"{% endblock %}
+
 {% block zone_other %}
 <foreignObject x="250" y="250" height="100" width="100">
   <img src="{{ url_for('static', filename='images/logo.png') }}" />
 </foreignObject>
+
+<foreignObject x="200" y="610" height="100" width="200">
+  <h2 style="text-align: center;">Robot Tokens</h2>
+</foreignObject>
+
+{{ input_robot_tokens_wider(10, 660, 0) }}
+{{ input_robot_tokens_wider(310, 660, 1) }}
+{{ input_robot_tokens_wider(310, 710, 2) }}
+{{ input_robot_tokens_wider(10, 710, 3) }}
+
+<foreignObject x="150" y="750" height="100" width="300">
+  <h2 style="text-align: center;">Token Totals</h2>
```

Comment:
[nit] I still think we should subtract one from the other and constrain it to positive integers, then call it "Tokens straddling zones".

I'm aware we lose a way of reading total counts, but that is handled by an error message anyway

Reply:
The challenge is that it's not just straddling zones -- a token could be "in" two (or more) robots too. So it's not really the count of the number of tokens which are in more than one place, but rather the number of extra times a token is seen.

Reply:
42a79e763e6af93fa7d5fc85502fce34a73c1586 hopefully makes this a bit more actionable while staying clear. Thoughts @Adimote